raise/transfer_rotorcraft_data.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
```

Comment:
I will eventually want to rewrite this in java to be honest. Liquibase will allow us to get rid of PHP, we can't allow ourselves to forswear one devil and immediately talk to another